common/lib/xmodule/xmodule/modulestore/__init__.py

```diff
@@ -22,11 +21,15 @@
 from .exceptions import InvalidLocationError, InsufficientSpecificationError
 from xmodule.errortracker import make_error_tracker
 from xmodule.assetstore import AssetMetadata
-from opaque_keys.edx.keys import CourseKey, UsageKey, AssetKey
+from opaque_keys.edx.keys import CourseKey, AssetKey
 from opaque_keys.edx.locations import Location  # For import backwards compatibility
 from xblock.runtime import Mixologist
 from xblock.core import XBlock
 
+# The below import is not used within this module, but ir is still needed becuase
+# other modules are imorting EdxJSONEncoder from here
+from openedx.core.lib.json_utils import EdxJSONEncoder
```

Comment:
Thanks.  You will probably need a pylint comment on the line:  `# pylint: disable=unused-import`

Reply:
Done

Reply:
A small point, but:  By putting the comment on the line before, you've disabled the warning for the entire file.  You should put it on the end of the line:
```
from openedx.core.lib.json_utils import EdxJSONEncoder            # pylint: disable=unused-import
```

Reply:
Fixed :+1: